src/lava/magma/core/learning/learning_rule.py

```diff
@@ -653,11 +762,11 @@ def __init__(
             x2_impulse=x2_impulse,
             x2_tau=x2_tau,
             y1_impulse=0,
-            y1_tau=2**32 - 1,
+            y1_tau=2**16 - 1,
```

Comment:
Can we replace these magic numbers (at least the 16?) with a constant or function?

Reply:
Replaced with `np.iinfo(np.int16).max`.